### PR TITLE
Add translations for Discover

### PR DESCRIPTION
**Pull request by pixeldesu**
One translation Pull Request coming right up! Handle as usual.

-4160591629798gchwy8x0

### CODE REVIEW

**Comment by ThatOneCalculator** [comment]
This🥴is😏so😮sussy😈👹I🤡think🤓Andy's🤪the☺️impostor📮📮📮📮

File: config/locales/views.en.yml [hunk 1]
```diff
@@ -106,6 +106,35 @@ en:
...
+        heading: "People"
+        description: "Newcomers and people who asked the most questions"
+        tab:
+          new: "New Users"
+          questions: "Active Askers"
+          answers: "Most Answers"
+    userbox:
+      new: "registered %{value} ago"
+      most:
+        zero: "answered 0 questions"
```
Comment (nilsding) [inline config/locales/views.en.yml]:
```suggestion
        zero: "answered no questions"
```

File: config/locales/views.en.yml [hunk 2]
```diff
@@ -106,6 +106,35 @@ en:
...
+        one: "answered 1 question"
+        other: "answered %{count} questions"
+      asked:
+        zero: "asked 0 questions"
```
Comment (nilsding) [inline config/locales/views.en.yml]:
```suggestion
        zero: "didn't ask any question"
```

**Review by raccube — APPROVED** [review]
By the way `git mv` will show that the file was moved in the diff.

**Pull request merged**